ts/store/middlewares/analytics.ts

```diff
@@ -263,6 +264,22 @@ const trackAction =
         }
         break;
       }
+      case getType(migrateToPaginatedMessages.request): {
+        return mp.track("MESSAGES_MIGRATION_START", {
+          total: action.payload.total
+        });
+      }
+      case getType(migrateToPaginatedMessages.success): {
+        return mp.track("MESSAGES_MIGRATION_SUCCESS", {
+          total: action.payload
+        });
+      }
+      case getType(migrateToPaginatedMessages.failure): {
+        return mp.track("MESSAGES_MIGRATION_FAILURE", {
+          failed: action.payload.failed,
+          succeeded: action.payload.succeeded
```

Comment:
```suggestion
          failed: action.payload.failed.length,
          succeeded: action.payload.succeeded.length
```

⚠️ in this way you will trace also the messages id
![Schermata 2022-03-30 alle 12 07 22](https://user-images.githubusercontent.com/822471/160807242-028f23c8-6a6f-4db1-b06b-faa4b73ebca2.png)